x-pack/plugins/upgrade_assistant/common/types.ts

```diff
@@ -5,8 +5,6 @@
  */
 
 import { SavedObject, SavedObjectAttributes } from 'src/core/public';
-// eslint-disable-next-line @kbn/eslint/no-restricted-paths
-import type { DeprecationInfo } from '../../../../src/core/server/elasticsearch/legacy/api_types';
```

Comment:
I am guessing these values are no longer being maintained on the core server side since it is in a "legacy" directory so they are safe to duplicate here without risk of something changing and TS not catching it?

Reply:
Correct. I think it's safe to duplicate for now. I did not see any support for it with the new client. I also think UA was the only app consuming the legacy type.